scripts/main.js

```diff
@@ -71,4 +71,29 @@ const departments = {
     }
 }
 
-console.log(departments);
\ No newline at end of file
+//opdracht 1
+console.log(departments);
+console.log("De afdeling Sales heeft", (departments.sales.numberOfEmployees), "medewerkers");
+console.log("Marketing is een leuke afdeling om te werken.", (departments.marketing.description));
+console.log("De afdeling Customer Service heeft", (departments["customer-service"].numberOfEmployees), "medewerkers");
+console.log("Sales is een uitdagende afdeling om te werken als Verkoopmanager.", (departments.sales.jobs[1]));
+
+//opdracht 2
+const userInput = prompt("Over welke afdeling wil je meer informatie? marketing / sales / customer-service").toLowerCase();
+console.log(userInput);
+
+if (departments[userInput]) {
+    console.log([userInput], "is een leuke afdeling om te werken. Er werken op dit moment", departments[userInput].numberOfEmployees, "medewerkers");
+} else {
+    console.error("ongeldige keuze")
+}
+
+//opdracht 3
+const title = prompt("Je koos"[userInput],"Over welke functie wil je meer weten? Voer een getal tussen 0 en 3 in");
+
+if (title >= 0 && title <= 3) {
```

Comment:
Ik denk dat er een foutje in de opdracht staat, want marketing heeft 5 functietitels. Dus dan zou het hier tot #4 moeten gaan.

Reply:
Haha, ja dat viel mij ook op idd. Ik heb het maar genegeerd. Thanks voor je feedback.